docs/scale_run_script.md

```diff
@@ -0,0 +1,71 @@
+# Using the scale run script
+
+The [utils/scale_run.sh](../utils/scale_run.sh) script is used to automate the
+execution of scale testing for the various EC2 instance types.  This script
+runs 5 rounds each of cold start and warm start scenarios.  It creates a
+directory for each run.
+
+Out of the box, it will run the following:
+
+* 1 run of 100 instances on a c5.large instance for the trial license
+  (10 hosts) scenario.  Tuning is set to true, but is not forced, so it will
+  not be applied to this instance due to insufficient ram.
+* 5 cold start runs (provisioning on the first) with tune set to true on the
+  following:
+    * c5.xlarge: Starting with 2000 instances
+    * c5.2xlarge: Starting with 4100 instances
+    * c5.4xlarge: Starting with 4100 instances
+* 5 warm start runs (using the provisioned hosts from the cold run above) with
+  tune set to true on the following:
+    * c5.xlarge: Starting with 1800 instances
+    * c5.2xlarge: Starting with 5000 instances
+    * c5.4xlarge: Starting with 6000 instances
+
+This behaviour is executed via the following command:
+```
+./util/scale_run.sh 2019.1.0
```

Comment:
If I understand it right, the "normal" way to run this is with the run-it set to the ticket number.  If so, having the first example a reader runs into being the one they should normally use would be helpful.

Reply:
I don't know if using the using the `--run-id` flag would be considered normal or not.   Regardless, my intention with the first example is to show the simplest way to invoke it.  The flags are described below and can then be appended by the user as needed.

Reply:
Well, like say we hire someone new.  And they get the job of running the scale tests.  What command line would you expect them to end up running... That should be the first command line example seen in the doc (IMO).  Even if it means adding a new small section saying so.  I am only assuming that would include the ticket number as the run-id.  If not, and this is what you would expect them to run, then all good.

Reply:
This is a great discussion.

The workflow I used was to create a dedicated test runner for running the slv-414 runs. So, I didn't need a run identifier to scope them for that effort.  For the re-run work that I did, a simple time stamp was enough to 1) Ensure that I didn't overwrite run data; 2) Allowed be to keep track of the sequence of runs.  In the current version of the script, a timestamp is always included to meet these criteria.

I am curious to hear @billclaytor thoughts on the matter.